Resources/Textures/Mobs/Animals/possum.rsi/meta.json

```diff
@@ -1,21 +1,35 @@
 {
   "version": 1,
-  "size":
-  {
+  "license": "CC-BY-SA-3.0",
+  "copyright": "New sprites taken from Baystation12 at https://github.com/Baystation12/Baystation12/blob/c7442716b52ce4edeef5eaf5d2fc7c57a08a3f01/icons/mob/simple_animal/possum.dmi, old sprites taken from goon https://github.com/goonstation/goonstation/commit/c98666f47a90d106c7a154cedffc747b3769e933. CC-BY-NC-SA-3.0",
```

Comment:
This should be a separate .rsi instead of mixing licenses in the copyright field like this